src/docs/customize/theming/forms.mdx

```diff
@@ -11,7 +11,7 @@ import { Playground } from 'docz'
 import { CheckboxField } from '../../../lib/components/ui/CheckboxField/CheckboxField'
 import { Radio } from '../../../lib/components/ui/Radio/Radio'
 import { SelectField } from '../../../lib/components/ui/SelectField/SelectField'
-import { TextField } from '../../../lib/components/ui/TextField/TextField'
+import TextField from '../../../lib/components/ui/TextField'
```

Comment:
I've met the same problem. We should use one or other variant in docs, but mixing them is not good idea. So we need to agree which variant is the correct.

Reply:
I think that outside the component root directory we should always import via `index.js` as that is the public API of the component.

Are there any reasons why importing directly from the component file is more practical?

Reply:
It is up to @adamkudrna. In my opinion, we should use index.js exports. The only part where raw/named export is necessary, it proptypes doc rendering.

Reply:
I see it is more complicated. Either way this change has no place in this PR. I'm reverting it.